src/builder/Actions.sol

```diff
@@ -837,6 +830,20 @@ library Actions {
         return (quarkOperation, action);
     }
 
+    function paymentMethodForPayment(PaymentInfo.Payment memory payment, bool useQuotecall)
```

Comment:
Great to pull this out into a separate helper. This can live in the `PaymentInfo` library.

Reply:
Also, there's some other functions in this file that this can be used in.

Reply:
moved into `PaymentInfo` and updated the other functions in `Actions` to use it